### PR TITLE
add install instructions

### PR DESCRIPTION
**Pull request by edgarriba**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?   
- [ ] Did you write any new necessary tests?  

## What does this PR do?
Fixes # (issue).

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 torch>=1.1
 pytorch-lightning>=0.7.1
-torchvision>=0.4.0
\ No newline at end of file
+torchvision>=0.4.0
```
Comment (williamFalcon) [inline requirements.txt:3]: Can we add these to the requirements-extra?
@Borda or does it make sense to allow these into reqs here?

i can imagine bolts getting bloated if we don't keep this slim? especially as we add submodules?

Comment (edgarriba) [reply requirements.txt:3]: which submodules will depend on vision stuff and which not ?

Comment (Borda) [reply requirements.txt:3]: I am thinking that we set as only requirements pytorch-lightning in setup file and all in requirements.txt are in fact extras...

Comment (Borda) [reply requirements.txt:3]: Altertive may be requirements.txt per submodules and in root link them together

Comment (williamFalcon) [reply requirements.txt:3]: but you auto read requirements.txt, so that won't work.

Comment (williamFalcon) [reply requirements.txt:3]: not every model here will be vision based (ie: audio, text, etc...). and we also have callbacks and such.
If someone only wants callbacks they don't need to install torchvision or anything image related.

Comment (edgarriba) [reply requirements.txt:3]: my guess is that you will have to have predefined requirements lists for each subggroup

Comment (justusschock) [reply requirements.txt:3]: yep, that way you could do something like `pip install bolts[vision]`

**Comment by Borda** [comment]
@edgarriba just rebased your PR, mind check that is it still correct with respect to large Bolts changes...
cc: @williamFalcon

**Pull request closed without merging**